code/modules/mob/mob_movement.dm

```diff
@@ -175,6 +175,9 @@
 	if(mob && .)
 		mob.throwing = 0
 
+	for(var/obj/O in mob)
+		O.on_mob_move(direct, src)
```

Comment:
We can probably get away with this, a test merge when this is closer to being ready will indicate if there are any issues, but it will unlock a lot of possibilities for people.